doc/api/globals.md

```diff
@@ -2,8 +2,15 @@
 
 <!-- type=misc -->
 
-These objects are available in all modules. Some of these objects aren't
-actually in the global scope but in the module scope - this will be noted.
+These objects are available in all modules. The following global-looking
```

Comment:
suggestion:

> The following variables may appear to be global but are not. They exist only in the scope of modules, see the module system documentation.

They would show up better in the TOC if each was listed in its original place instead of this bullet list, but with the docs replaced with a explanation and a link to the module system docs.